embulk-input-mysql/src/test/java/org/embulk/input/mysql/BasicTest.java

```diff
@@ -41,6 +42,7 @@ private static String readResource(String fileName)
     @Before
     public void setup()
     {
+        TimeZone.setDefault(TimeZone.getTimeZone("Europe/Helsinki")); // +200
```

Comment:
I was missing the settings included in https://github.com/embulk/embulk-input-jdbc/pull/94 and https://github.com/embulk/embulk-input-jdbc/pull/95.